src/recommendations/RecommendationsPage.jsx

```diff
@@ -24,6 +24,7 @@ const RecommendationsPage = (props) => {
 
   const [isLoading, setIsLoading] = useState(true);
   const [recommendations, setRecommendations] = useState([]);
+  const [algoliaRecommendations, setAlgoliaRecommendations] = useState([]);
```

Comment:
Lets use boolean value instead of saving the whole list in two places. 
```suggestion
  const [isAlgoliaRecommendation, setIsAlgoliaRecommendations] = useState(false);
```

Reply:
Set `isAlgoliaRecommendation` only when we get the recommendations from Algolia and then use it in the useEffect along with already existing `recommendations` state.